src/Calculator.tsx

```diff
@@ -65,29 +65,29 @@ class Calculator extends React.Component {
         return (
             <div>
               <h1>Confusion Matrix</h1>
-              <table border="1">
+              <table style={{borderWidth: 1}}>
                 <tr>
-                    <td colSpan="2"></td><td colSpan="2" align="center">Reality</td><td>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</td>
+                    <td colSpan={2}></td><td colSpan={2} align="center">Reality</td><td>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</td>
                 </tr>
                 <tr>
-                    <td colSpan="2"></td><td><b>Positive</b></td><td><b>Negative</b></td><td>&nbsp;</td>
+                    <td colSpan={2}></td><td><b>Positive</b></td><td><b>Negative</b></td><td>&nbsp;</td>
                 </tr>
                 <tr>
-                    <td rowspan="2">Reported</td>
+                    <td rowSpan={2}>Reported</td>
```

Comment:
> src/Calculator.tsx:73:25 - error TS2322: Type '{ children: string; rowspan: string; }' is not assignable to type 'DetailedHTMLProps<TdHTMLAttributes<HTMLTableDataCellElement>, HTMLTableDataCellElement>'.
>   Property 'rowspan' does not exist on type 'DetailedHTMLProps<TdHTMLAttributes<HTMLTableDataCellElement>, HTMLTableDataCellElement>'. Did you mean 'rowSpan'?

Same as before, except you also used `rowspan` instead of `rowSpan`. I'm not sure exactly how much of an impact this has on React (it still works, after all), but again, React is complaining about it.